Add `SVS` and `SVS2` contest articles (#7450)
ppy/osu-wiki: wiki/Contests/en.md

### osu!taiko Featured Artist Cup

> The **osu!taiko Featured Artist Cup** is an osu!taiko mapping contest where contestants are simply asked to map a song from one of osu!'s Featured Artists. The contest can sometimes have specific limitations/gimmicks in order to put the contestants' skills to the test.

Reviewer: ```suggestion
The **osu!taiko Featured Artist Cup** is an osu!taiko mapping contest where contestants are simply asked to map a song from one of osu!'s Featured Artists. The contest sometimes has specific limitations/gimmicks in order to put the contestants' skills to the test.
```

| Contest No. | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] | Results |
| :-: | :-- | :-- | :-- | :-- |
| [#1](/wiki/Contests/o!tFAC/1) | ![][flag_KR] [woosungko](https://osu.ppy.sh/users/14184157) | ![][flag_TH] [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | ![][flag_IT] [D3kuu](https://osu.ppy.sh/users/7807444) | [#1](https://osu.ppy.sh/community/forums/topics/1520800) |
| [#2](/wiki/Contests/o!tFAC/2) | *TBD* | *TBD* | *TBD* | [#1](https://osu.ppy.sh/community/forums/topics/1588200) |

### Slider Velocity Showdown

> The **Slider Velocity Showdown** is the osu!taiko mapping contest with a focus on creativity and innovation. This contest is open to anyone who wanted to flex their creative muscles, whether they have never placed an inherited point in their life or if they have been mapping for years.

Reviewer: ```suggestion
The **Slider Velocity Showdown** is an osu!taiko mapping contest with a focus on creativity and innovation. This contest is open to anyone who wants to flex their creative muscles, whether they have never placed an inherited point in their life or if they have been mapping for years.
```
